go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -1113,6 +1130,51 @@ func setBridgeOfPorts(bridge *bridgeConfiguration) error {
 	return nil
 }
 
+// initSvcViaMgmPortRoutingRules creates the svc2managementport routing table, routes and rules
+// that let's us forward service traffic to ovn-k8s-mp0 as opposed to the default route towards breth0
+func initSvcViaMgmPortRoutingRules(hostSubnets []*net.IPNet) error {
+	// create ovnkubeSvcViaMgmPortRT and service route towards ovn-k8s-mp0
+	for _, hostSubnet := range hostSubnets {
+		isIPv6 := utilnet.IsIPv6CIDR(hostSubnet)
+		gatewayIP := util.GetNodeGatewayIfAddr(hostSubnet).IP.String()
+		for _, svcCIDR := range config.Kubernetes.ServiceCIDRs {
+			if isIPv6 == utilnet.IsIPv6CIDR(svcCIDR) {
```

Comment:
does it make sense to add an error for the case where AF for hostsubnet and svcCIDR is different ?

Reply:
hmm I think no since eventually one of them will match and if they don't then we can't do anything about it.
Example if its ipv4, then isIPv6 will be false and utilnet.IsIPv6CIDR(svcCIDR) will also be false and vice versa as both will be true for IPv6. It will be noise to log these in dualstack env's.

Reply:
yeah agree for dual-stack will generate noise for no reason